README.md

```diff
@@ -62,124 +62,6 @@ make dev
 
 Read more about Rails Application Template in the [official Rails Guides](https://guides.rubyonrails.org/rails_application_templates.html).
 
```

Comment:
```suggestion

See the contribution guidelines in the [Wiki](../../wiki).
```

Should we keep a small mention of the wiki in the ReadMe file? 🤔

Reply:
Thanks 🙏🏼 . Fixed in 45ceeed0eaa8e94885f7135afe170258845f71a1